InnerEye/ML/plotting.py

```diff
@@ -303,6 +306,10 @@ def plot_contours_for_all_classes(sample: Sample,
         if class_index == 0:
             continue
         ground_truth = sample.labels[class_index, ...]
+
+        if np.isnan(ground_truth[0, 0, 0]):
```

Comment:
This check is now done at two places in the code, meaning it can run out of sync. Refactor into a method "is_missing_ground_truth"?

Reply:
That wasn't my code but yes, agreed. Fixed.